src/common/translations/localize.ts

```diff
@@ -38,12 +36,19 @@ if (shouldPolyfill()) {
  * }
  */
 
-export const computeLocalize = (
+export const computeLocalize = async (
   cache: any,
   language: string,
   resources: Resources,
   formats?: FormatsType
-): LocalizeFunc => {
+): Promise<LocalizeFunc> => {
+  if (!polyfillLoaded && shouldPolyfill()) {
+    await import("@formatjs/intl-pluralrules/polyfill-locales");
+    polyfillLoaded = true;
+  } else if (!polyfillLoaded) {
+    polyfillLoaded = true;
+  }
```

Comment:
Now we're doing 2 if-checks, even if polyfills is loaded or not needed. Cleaner:
```suggestion
  if (!polyfillLoaded) {
    if (!shouldPolyfill()) {
      await import("@formatjs/intl-pluralrules/polyfill-locales");
    }
    polyfillLoaded = true;
  }
```